pkg/pub_integration/test/fake_pub_server_test.dart

```diff
@@ -37,12 +39,38 @@ void main() {
     tearDownAll(() async {
       await tempDir?.delete(recursive: true);
       await fakePubServerProcess?.kill();
+      httpClient.close();
     });
 
     test('standard integration', () async {
+      final inviteUrlFuture = fakePubServerProcess
+          .waitForLine((line) => line.contains('/admin/confirm/new-uploader/'));
```

Comment:
You're fake email provider writes emails to stdout, right?

Reply:
yes, that's correct